de.bund.bfr.knime.js/src/js/app/app.simulation.js

```diff
@@ -660,7 +831,18 @@ class APPSimulation {
 					}
 					// change other inputs
 					else {
-						! _isNull( paramValue ) ? field.input.val( paramValue ) : null;
+					    if(field.param.dataType == 'FILE'){
+					    	
+					    	let paramId  = field.input.attr('id').split('_').pop();
+					    	let filesContainer = $('#filesArea-'+paramId);
+					    	let fileElement = $("<div ><hr/><p>"+ paramValue +"</div>");
+				    		
+                            
+		                    filesContainer.html(fileElement);
+						}
```

Comment:
@ahmadswaid this piece of code (line 834-842) does not work right now. The parameter values are not shown, regardless Windows or Linux. They are also removed from the simulation when clicking close and apply. Model taken for this is: https://knime.bfr.berlin/landingpage/DB/download/0f454533-c324-43a4-a1d1-5a997125a1ba?status=Any&repository=Any

<img width="783" alt="image" src="https://user-images.githubusercontent.com/38949263/212682298-b4287cfa-8e88-49c5-86c5-5f1012e7155e.png">
<img width="805" alt="image" src="https://user-images.githubusercontent.com/38949263/212682351-a283e442-f307-4170-ba4b-070a01ef3792.png">